genie-core/src/main/java/com/netflix/genie/core/jpa/services/JpaJobSearchServiceImpl.java

```diff
@@ -362,4 +363,16 @@ public String getJobHost(@NotBlank final String jobId) throws GenieException {
             throw new GenieNotFoundException("No job execution found for id " + jobId);
         }
     }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public Long getActiveJobCountForUser(@NotBlank final String user) throws GenieException {
+        log.debug("Called for jobs with user {}", user);
+        if (StringUtils.isEmpty(StringUtils.trimToEmpty(user))) {
+            throw new IllegalArgumentException("User cannot be null or empty");
+        }
+        return this.jobRepository.countJobsByUserAndStatusIn(user, JobStatus.getActiveStatuses());
```

Comment:
Rather than returning the Nullable `Long` you might want to check for null here and dereference to the primitive  `long` or `0L` if null that way clients to this service don't have to worry about performing the null check themselves.